packages/harmony/.storybook/preview.ts

```diff
@@ -15,25 +15,24 @@ import 'foundations/color/semantic.css'
 import 'foundations/motion/motion.css'
 import 'foundations/corner-radius/corner-radius.css'
 import 'foundations/shadows/shadows.css'
-import { darkTheme, lightTheme } from './theme'
 import { themes } from '../src/foundations/theme'
 import { HarmonyDocsContainer } from './docs'
 import { ThemeProvider } from '@emotion/react'
 
 // Default to docs view
 export const parameters = {
   viewMode: 'docs',
-  darkMode: {
-    light: lightTheme,
-    dark: darkTheme
-  },
+  enableShortcuts: false,
   docs: {
     container: HarmonyDocsContainer,
     toc: true,
     components: {
       ComponentRules,
       RelatedComponents
     }
+  },
+  options: {
+    enableShortcuts: false
```

Comment:
do you need this twice?

Reply:
no good call, these actually didnt even work